CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added traits for updating / mapping the pixels of grid types. Also added a trait for creating grid types using a function for each pixels location.
```

Comment:
"Added methods"? You implemented the functionality too

Reply:
[80d8a47](https://github.com/geo-engine/geoengine/pull/561/commits/80d8a4701b42eec34ec0d075b9523292a48c9b6f)